### DISCUSSION TITLE
嘿嘿，摸进治安局了，让我康康~

### DISCUSSION THREAD

**Discussion by ENDer-Z1** [share121/inter-knot#2108] [其他]
![20240814193206](https://github.com/user-attachments/assets/a60cef7d-c91a-4541-a0de-9eb6bb4dfafa)
这年头，治安官都开始上班摸鱼了，几个文件夹拖来拖去的。
关注我，持续带来.....
“欸，干什么的！敢在治安局里面偷拍，拷走！”
“不不不，我不是我....我没有呀！”
——使用语音输入，已自动上传绳网。

**Comment by poolofwater** [comment]
哎，蛐蛐绳匠还敢进治安局，不是自投罗网吗<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Genshin-Neuvillette** [comment]
菜鸟绳匠是这样的，治安局只要考虑抓人就可以了，而菜鸟绳匠考虑的就要多了
